packages/sdk/scripts/test_deposit.ts

```diff
@@ -0,0 +1,60 @@
+import { ethers } from "ethers";
+import { formatEther } from "ethers/lib/utils";
+import { ServiceBridge } from '../src/service_bridge';
+import { delay } from "../src/utils";
+
+async function main() {
+
+    // TODO: put me in a .env file
+    const l1Url = 'http://localhost:8545'
```

Comment:
can you just use dotenv to load a testing env file like `.env.test`?

Reply:
sounds good